### ISSUE TITLE
Where is GCD 7 Tool bar after installing?

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/gcd#335]
## The Problem
From user:
> I  have installed GCD 7 in my ArcGis 10.6 and when I try to add the toolbars of GCD7, althouh I can see they are selected, they do not appear ... Do you know what can be the problem?

**Comment by joewheaton** [comment]
> Yes, I did try this but it did not work :-( I re-installed the add-in but it does not work ... any idea in what I can do?

Interesting... why don't your respond to that GitHub post, and maybe record a little YouTube video of what you're trying and what's happening?

**Comment by joewheaton** [comment]
Damia, I'm also thinking that perhaps the toolbar is installed, but perhaps you were connected on a laptop with multiple monitors and sometimes these toolbars can get lost on an extended desktop space... Just a thought.

**Comment by philipbaileynar** [comment]
Damia,

Thanks for the video. Is this one very wide screen or did the video just capture two monitors?

If this is one very wide screen then this might be part of the problem. I think that the GCD is installed OK and that the GCD toolbar is getting hidden or misplaced on the screen somehow. ArcGIS does not deal with high resolution displays very well. My recommendation:

1. Change your display to a single monitor at 1080p (1960x1080).
1. Close ArcMap.
1. Open ArcMap.
1. Hide the GCD toolbar (uncheck the tickbox beside the Geomorphic Change Detection 7 item in the toolbar dropdown menu).
1. Turn on the Geomorphic Change Detection 7 item in the toolbar dropdown menu.
1. If you see the toolbar then dock it in the main toolbar area at the top of ArcMap.
1. Revert to your previous display resolution.

There is a small chance that performing these steps at a more "normal" display resolution might reset the screen coordinates where ArcMap wants to display the GCD toolbar.

**Comment by damiavericat** [comment]
Hi Philip

Thank you very much for your help. I have tried but with no luck ... see the new video I have attached here 

https://youtu.be/ZE_Wl9wMiBY

Any idea of what I can do? 

Thank you very much
Best
Damià

**Comment by joewheaton** [comment]
Hi @damiavericat. From both your videos, it is clear that the GCD toolbar is installed. I suspect that for some reason, your registry from an old installation of GCD has the toolbar somewhere off the current workspace. The easiest thing is to just see if you notice the toolbar hiding on your other monitor somewhere. Make sure all the other applications are minimized and just have ArcGIS open. Do you see it anywhere. Fiddling with the resolution of the screen might help too.  However, you may have to dig deeper and go into your Registry and change the settings. 

See: 
https://dragons8mycat.com/2014/02/12/gis-tips-arcgis-i-lost-my-toolbar-and-other-toolbar-problems/
https://community.esri.com/ideas/4831

@philipbaileynar, I'm not sure what else to suggest. I don't think a reinstall or ESRI fixes this unless you do a complete clean uninstall with something like iObit Uninstaller, which actually gives you option to delete all the registry keys (otherwise they're left there). I'm not sure the consequences of deleting the registry keys manually?

**Comment by philipbaileynar** [comment]
This could also be helpful:

https://support.esri.com/en/technical-article/000011505

**Comment by damiavericat** [comment]
Hi Joe and Philip! 

thank you very much for your help. I have found the solution; of course, thanks to you and your indications. I have tried several of the options you have indicated and finally what have worked out well has been what is suggested in the link: https://support.esri.com/en/technical-article/000011505

Mainly: 

A. Close ArcCatalog and ArcMap
B. Navigate to root C:\Users\CURRENT_USERNAME\AppData\Roaming\ESRI\Desktop10.0\ArcMap\Templates

Note:
The Application Data folder may be hidden. If so, go to Start > Control Panel > Folder Options. Click the View tab and select 'Show hidden files, folders, and drives'.

C. Open the Templates folder
D. Right-click the file Normal.mxt file and rename it. For example, Normalold.mxt. Re-start ArcMap.

So, all solved for now … Again, thank you very much for all
Best
Damià
